Hashtable.c

```diff
@@ -10,7 +10,9 @@ in the source distribution for its full text.
 #include "Hashtable.h"
 
 #include <assert.h>
+#include <stddef.h>
 #include <stdint.h>
+#include <inttypes.h>
```

Comment:
Header ordering is alphabetical:
```suggestion
#include <inttypes.h>
#include <stddef.h>
#include <stdint.h>
```